debian/control

```diff
@@ -4,9 +4,8 @@ Priority: optional
 Maintainer: Tobias Hansen <tobias.han@gmx.de>
 Build-Depends: debhelper (>= 8.0.0),
                pkg-config,
-               liballegro4-dev,
-               libaldmb1-dev,
                libfreetype6-dev,
+               libsdl2-dev (>= 2.0.9),
```

Comment:
is there a specific reason you request the >= 2.0.9 version of SDL2 ?

Reply:
SDL2_sound won't work with lower versions of SDL2. This means it could work in systems with Debian Buster and beyond but won't work with system SDL2 in Debian Jessie. Feel free to test.